Command/RemoveThumbsCommand.php

```diff
@@ -238,4 +234,14 @@ protected function log($message)
             $this->output->writeln($message);
         }
     }
+
+    /**
+     * Get question helper or the dialog helper for SensioGeneratorBundle 2.3/2.4 compatibility.
+     *
+     * @return QuestionHelper
+     */
+    private function getQuestionHelper()
```

Comment:
Please add return type hinting and remove the `@return` comment

Reply:
Do you mean php7-style (`private function getQuestionHelper(): QuestionHelper`)?
Should I do the same with other functions?

Reply:
Yes, absolutely please.